csrankings-t.csv

```diff
@@ -879,16 +879,14 @@ Torsten Höfler,ETH Zurich,https://htor.inf.ethz.ch,DdBvcBEAAAAJ
 Torsten Ingo Braun,University of Bern,http://www.iam.unibe.ch/~braun,eL8OzgIAAAAJ
 Torsten Kroeger,Karlsruhe Institute of Technology,http://www.ipr.kit.edu/mitarbeiter_2538.php,bfec5vAAAAAJ
 Torsten Kröger,Karlsruhe Institute of Technology,http://www.ipr.kit.edu/mitarbeiter_2538.php,bfec5vAAAAAJ
-Torsten Kuhlen,RWTH Aachen University,http://www.vr.rwth-aachen.de/person/1/,O9ywg_UAAAAJ
 Torsten Möller,University of Vienna,https://cs.univie.ac.at/torsten.moeller,3cGVmqsAAAAJ
 Torsten Muetze,University of Warwick,https://warwick.ac.uk/fac/sci/dcs/people/torsten_mutze,9232pxwAAAAJ
 Torsten Mütze,University of Warwick,https://warwick.ac.uk/fac/sci/dcs/people/torsten_mutze,9232pxwAAAAJ
 Torsten Sattler,Czech Technical University,https://tsattler.github.io,jzx6_ZIAAAAJ
 Torsten Schaub,University of Potsdam,https://www.cs.uni-potsdam.de/wv/people/torsten,XknBv7MAAAAJ
 Torsten Söderström,Uppsala University,http://user.it.uu.se/~ts,NOSCHOLARPAGE
 Torsten Suel,New York University,http://engineering.nyu.edu/~suel,eQUn8ugAAAAJ
-Torsten W. Kuhlen,RWTH Aachen University,http://www.rwth-aachen.de/cms/root/Die-RWTH/Jobs-Ausbildung/Berufungen-an-der-RWTH-Aachen/Neuberufene/~kgfh/Univ-Prof-Torsten-Wolfgang-Kuhlen/lidx/1,O9ywg_UAAAAJ
-Torsten Wolfgang Kuhlen,RWTH Aachen University,http://www.rwth-aachen.de/cms/root/Die-RWTH/Jobs-Ausbildung/Berufungen-an-der-RWTH-Aachen/Neuberufene/~kgfh/Univ-Prof-Torsten-Wolfgang-Kuhlen/lidx/1,O9ywg_UAAAAJ
+Torsten W. Kuhlen,RWTH Aachen University,http://www.vr.rwth-aachen.de/person/1/,O9ywg_UAAAAJ

```

Comment:
Both of these must be listed:
* Torsten W. Kuhlen 
* Torsten Wolfgang Kuhlen

Reply:
incorporated. Thanks for the pointer.